docker-compose.yml

```diff
@@ -867,13 +867,15 @@ services:
       args:
         base: ${R_ORG}/${R_IMAGE}:${R_TAG}
         r_dev: ${ARROW_R_DEV}
+        devtoolset_version: ${DEVTOOLSET_VERSION}
     shm_size: *shm-size
     environment:
       LIBARROW_DOWNLOAD: "false"
       ARROW_HOME: "/arrow"
       ARROW_R_DEV: ${ARROW_R_DEV}
       # To test for CRAN release, delete ^^ these two env vars so we download the Apache release
       ARROW_USE_PKG_CONFIG: "false"
+      devtoolset_version: ${DEVTOOLSET_VERSION}
```

Comment:
I _think_ this needs to be capitalized, it's put in the env verbatim

Reply:
Yeah, probably. I'm going try without on crossbow to confirm while it's here and then send the push to fix it